extensions/wikia/JsonFormat/JsonFormatBuilder.php

```diff
@@ -40,7 +40,8 @@ public function popInfoboxKeyValue( JsonFormatInfoboxKeyValueNode $keyValueNode
 			array_pop($this->jsonStack);
 			$this->currentContainer = $this->jsonStack[sizeof($this->jsonStack)-1];
 		} else {
-			throw new JsonFormatException('Invalid infobox');
+			# log invalid instead of failing
+			\Wikia\Logger\WikiaLogger::instance()->debug('Invalid infobox');
```

Comment:
I went with debug, because we don't use infobox output anywhere (snippets or simple json output).